Tablas.py

```diff
@@ -35,6 +35,10 @@ def buscarZ(x):
     <param name="n">Grados de libertad.</param>
     <return>Valor de la tabla JiCuadrada.</return>
 """
+
+def buscarT(alpha, n):
```

Comment:
```suggestion
"""
    <summary>
        Documentacion
    </summary>
"""
def buscarT(alpha, n):
```

Falta la documentacion, e inclusive tiene que ser en su propio lado, la documentacion de arriba es de la JiCuadrada, no de la tablaT

Reply:
Asi mismo, en la linea 32 dice
`Funcion que calcula el valor de la tabla Z.`

Podrias cambiarlo a la JiCuadrada, porfa

Reply:
```suggestion
"""
    <summary>
        Función que calcula el valor de la tabla T de Student.
    </summary>
    <param name="alpha">Valor de la significancia.</param>
    <param name="n">Grados de libertad.</param>
    <return>Valor de la tabla T.</return>
"""

def buscarT(alpha, n):
    return tablaT[alpha][n-1]   
```
Documentación de la función de T realizada, además corregí el cambio que me pediste de la Ji Cuadrada e hice correcciones ortográficas a la documentación.